pkg/generators/backend/cron_deployment.go

```diff
@@ -44,9 +43,6 @@ func (gen *CronGenerator) Deployment(hashErrorMonitoring string) basereconciler.
 				Template: corev1.PodTemplateSpec{
 					ObjectMeta: metav1.ObjectMeta{
 						Labels: gen.LabelsWithSelector(),
-						Annotations: map[string]string{
-							saasv1alpha1.RolloutTriggerAnnotationKeyPrefix + config.ErrorMonitoringSecretName: hashErrorMonitoring,
-						},
```

Comment:
This is now done automatically by the basereconciler itself given a list of RolloutTriggers